changelog.txt

```diff
@@ -1,3 +1,18 @@
+13.0-beta2 (date TBD)
+------------------------------------------------------------------------
+Feature: Multi-track level crossings (#9931)
+Feature: Add cargo filter support to vehicle list. (#8308)
+Feature: Display power-to-weight ratio in ground vehicle details GUI (#10123)
+Feature: Add buttons to toggle music in the Game Options menu (#9727)
+Feature: Automatic console command screenshot numbering with a filename ending in '#' (#9781)
+Change: Let AI developers edit non-editable AI/Game Script Parameters (#8895)
+Change: Allow building docks on clearable watered object tiles (#8514)
+Fix #10011: Incorrect infrastructure totals when overbuilding bay road stop (#10143)
+Fix #10023: Allow negative input in text fields when needed (#10112)
+Fix #10117: Object burst limit allowed one fewer object than the setting (#10120)
+Fix #9908: Close company windows only after all its infrastructure is removed (#9912)
```

Comment:
The line for #9908/#9912 is fairly mystifying unless you refer back to the issue which is being fixed.
Might it be worth re-phrasing it in terms of the problem which was fixed (i.e. "Fix crash which could occur when a company was deleted when a depot window was open" or similar)?